spec/services/unsuspend_account_service_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe UnsuspendAccountService, type: :service do
-  shared_examples 'common behavior' do
+  shared_context 'with common behavior' do
```

Comment:
It might be clearer to rename it to “with common context”

Reply:
rebased and changed that wording.